app/models/describe_job.rb

```diff
@@ -0,0 +1,79 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# TODO: Maybe move other items from DelayedJob initializer code into here?
+class DescribeJob < OpenStruct
```

Comment:
does it need to inherit from openstruct?
```suggestion
class DescribeJob
```

Reply:
also, "DescribeJob" doesn't really read as a noun. Perhaps "JobDetails" or "JobInfo"?